bin/identify_dormant_github_users.py

```diff
@@ -80,7 +80,10 @@ def download_github_dormant_users_csv_from_s3():
 
 def get_dormant_users_from_github_csv(github_service: GithubService) -> list:
     """
-    This function depends on a preliminary manual process: a GitHub Enterprise user must first download the 'dormant.csv' file from the GitHub Enterprise dashboard and then upload it to the 'operations-engineering-dormant-users' S3 bucket. Once this setup is complete, this function will download the 'dormant.csv' file from the S3 bucket and extract a list of usernames from the file. This process ensures that the most current data regarding dormant users is used.
+    This function depends on a preliminary manual process: a GitHub Enterprise user must first download the 'dormant.csv'
+    file from the GitHub Enterprise dashboard and then upload it to the 'operations-engineering-dormant-users' S3 bucket.
+    Once this setup is complete, this function will download the 'dormant.csv' file from the S3 bucket and extract a list of 
```

Comment:
I believe it's complaining about the space at the end 👀 

```suggestion
    Once this setup is complete, this function will download the 'dormant.csv' file from the S3 bucket and extract a list of
```